.github/workflows/.github-actions.yml

```diff
@@ -0,0 +1,30 @@
+name: pyRACF package
```

Comment:
I would change this to the following. This field is going to be what is displayed on the GitHub actions build badge. We are not really packaging anything here. We are just running the linters and running the unit tests.

```yaml
name: pyRACF Linting & Unit Test
```

Reply:
This makes sense, agree and implemented as suggested.

Reply:
Looks good.